### DISCUSSION TITLE
PARTICLES Extension - Particle Contact and Radius

### DISCUSSION THREAD

**Discussion by CbsTau** [ProjectPhysX/FluidX3D#245] [Q&A] [answered]
Hello Dr. Lehmann,

The 2-way coupled particles have density and react to gravity and the flow field, and also interact with TYPE_S walls. Could you clarify the following:

1. Do the particles behave as a spherical particle with a radius? If not, how are the forces on the particle determined? If there is a radius, can it be specified by the user?

2. Are the particles able to interact with each other? Do they make contact with other particles or can multiple particles occupy the same physical space, and pass through each other?

3.  Do the particles affect the flow field? Can the particles cause blocking or clogging of the flow?

4. What are the interaction properties between the particles themselves, and the TYPE_S walls? Are collisions perfectly plastic, or can a coefficient of restitution be considered?

I guess all these questions highlight the need for a more detailed "Theory Guide" which explains a bit more of the physics which are being included or excluded in the solver.  The answers to these kinds of questions should be collated into a Theory guide, alongside the current Documentation which is more of a User guide / Getting Started guide.

Many thanks in advance,
Clinton.

**Comment by ProjectPhysX** [comment, marked as answer]
Hi @CbsTau,

1. Particles are mathematical point-particles without size. However they have a hydrodynamic interaction radius of 0.5 (1/2 the LBM lattice constant). See [here](https://doi.org/10.1186/s43591-021-00018-8) in the [Supplementary Information](https://static-content.springer.com/esm/art%3A10.1186%2Fs43591-021-00018-8/MediaObjects/43591_2021_18_MOESM1_ESM.pdf) for details.
2. Particles do not directly interact with each other. There is no direct force between particles, nor a collision algorithm. But they can interact with each other by creating a flow field which moves other particles.
3. Yes particles affect the flow field through buoyancy. Many particles together can sink like sand or float like bubbles, and thereby create a flow field that moves other particles in 2-way interaction. Since particles cannot collide with each other, and since they are directly advected with the flow velocity (no friction-slip model), they cannot cause clogging.
4. Interaction with `TYPE_S` walls is a hard potential to force particles out of the wall and keep them in the fluid. Collisions are fully inelastic.

So the particle model extension is rather rudimentary - expand it as needed for your applications.
Note that particles are just an extension, and the LBM solver itself doesn't need particles to model Navier-Stokes.

Kind regards,
Moritz
